lib/eslintBaseConfig.json

```diff
@@ -39,9 +39,10 @@
       {
         "groups": [
           ["builtin", "external"],
-          ["internal", "unknown", "parent", "sibling", "index"],
+          ["index", "internal", "parent", "sibling"],
           ["type"],
-          ["object"]
+          ["object"],
+          ["unknown"]
```

Comment:
The [documentation for the `order` rule](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) added examples of all group types except for `unknown`, so now that it's clear what each group type does, rather than have `unknown` mixed into the `Internal Imports` group, move it to its own group. Skimming through the ESLint plugin code doesn't reveal any obvious details either.